addons/mail/static/src/discuss/voice_message/common/voice_analyser.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.VoiceAnalyser" owl="1">
+        <div class="d-flex align-items-center my-1 position-relative">
+            <t t-set="playOrPause"><t t-if="state.playing">Pause</t><t t-else="">Play</t></t>
+            <button class="border-0 btn rounded-pill fa ms-2 bg-100"
+                t-att-class="state.playing ? 'fa-stop text-danger btn-sm' : 'fa-play btn-light btn-sm'"
+                t-att-title="playOrPause" t-on-click.stop="!props.attachment.uploading ? () => this.soundEffectsService.onClickAnalyser(this) : ''"/>
```

Comment:
Could show `fa-repeat` icon when reaching the end, so it's clearer it goes back to the start